README.md

```diff
@@ -3,6 +3,10 @@ My App
 
 TODO: Include a summary of app functionality and a screenshot
 
+Pull Requests
+===
+To specify the semantic version bump of your pull request, choose the `major-version` for a major version update, `minor-version` for a minor version update, and none for a patch update.
```

Comment:
This probably needs to be a bit more verbose. How about:

---

Every app deployment requires that the `version` property in the `manifest.json` file be updated to reflect the new app version. This is so Deskpro can detect changes and add/upgrade apps accordingly. As such, we've made changing versions easy by having CI make the actual version change for you. Here's what we do:

* We increment `patch` versions, i.e. `1.0.1`, automatically. You don't need to do anything and this will happen
* Minor versions, i.e. 1.1.0, are incremented if you add the `minor-version` GitHub label to your PR
* Major versions, i.e. 2.0.0, are incremented if you add the `major-version` GitHub label to your PR